### PR TITLE
Added decision to use existing API

### PR DESCRIPTION
**Pull request by dendle**
# ADR Record for DQT API decision
The Find My TRN application will use the DQT API to access data in the DQT.

### CODE REVIEW

File: docs/architecture/decisions/0003-use-existing-DQT-API.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+We need a way of accessing data in the DQT - there are a few options.  Either integrate with the DQT directly, create an new API or use the existing DQT API (v3) by using/extending its functionality.
+
+## Decision
+
+We will use the existing DQT API, and extend it where necessary to support the features of the Find My TRN service
+
+## Consequences
+
+* We dont have to write an API from scratch
+* We are dependant on the DQT API team to make any changes
```
Comment (benilovj) [inline docs/architecture/decisions/0003-use-existing-DQT-API.md:20]:
```suggestion
* We are dependent on the DQT API team to make any changes
```

**Review by benilovj — APPROVED** [review]
LGTM barring typo

**Comment by benilovj** [comment]
@dendle can I suggest we close this PR and move the ADR to https://github.com/DFE-Digital/find-my-trn ?

**Comment by benilovj** [comment]
cc @tvararu

File: docs/architecture/decisions/0003-use-existing-DQT-API.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We need a way of accessing data in the DQT - there are a few options.  Either integrate with the DQT directly, create an new API or use the existing DQT API (v3) by using/extending its functionality.
+
+## Decision
+
+We will use the existing DQT API, and extend it where necessary to support the features of the Find My TRN service
+
+## Consequences
+
+* We dont have to write an API from scratch
```
Comment (defong) [inline docs/architecture/decisions/0003-use-existing-DQT-API.md:19]:
```suggestion
* We don't have to write an API from scratch
```

Comment (benilovj) [reply docs/architecture/decisions/0003-use-existing-DQT-API.md:19]: Close but no cigar @defong https://smartquotesforsmartpeople.com/

**Pull request merged**